doc/maintaining/installing/install-from-docker-compose.rst

```diff
@@ -3,28 +3,22 @@
 ===================================
 Installing CKAN with Docker Compose
 ===================================
-This chapter is a tutorial on how to install the latest CKAN (master or any stable version)
-with Docker Compose. The scenario shown here is one of many possible scenarios and environments
-in which CKAN can be used with Docker.
+This chapter is a tutorial on how to install CKAN with Docker Compose. The scenario shown here is 
+one of many possible scenarios and environments in which CKAN can be used with Docker.
 
 This chapter aims to provide a simple, yet fully customizable deployment - easier to configure than
 a source install, more customizable than a package install.
 
 The discussed setup can be useful as a development / staging environment; additional care has to be
 taken to use this setup in production.
 
-.. note:: Some design decisions are opinionated (see notes), which does not mean that the
-  alternatives are any worse.
-  Some decisions may or may not be suitable for production scenarios, e.g. the use of CKAN master.
-  Notably, this tutorial does not use Docker Swarm; additional steps may need to be taken to adapt
-  the setup to use Docker Swarm.
 
 --------------
 1. Environment
 --------------
-This tutorial was tested on Ubuntu 20.04 LTS.
-The hosts can be local environments or cloud VMs. It is assumed that the user has direct access
-(via terminal / ssh) to the systems and root permissions.
+The hosts can be local environments or cloud VMs. This tutorial works perfectly on Ubuntu 20.04 LTS (Focal) 
+and MacOS 11.5 (Big Sur). Other hosts should be fine also. It is assumed that the user has direct access 
+(via terminal / ssh) to the systems and has root access.
```

Comment:
I personally think that the "a. Storage" section below is out of scope for this repo and I'd lean towards deleting it. If you think that's useful we can keep it in another location